participants/sanjitcodes.md

```diff
@@ -2,4 +2,19 @@ Name: SANJIT ANAND
 GitHub username: sanjitcodes
 University: NIT, Surat
 Year: 1st
-Branch: Electronics and Communication
\ No newline at end of file
+Branch: Electronics and Communication Engineering
+
+## I am from Amritsar, Punjab.
+
+# **HOBBIES**
+1. Lawn Tennis
+2. Cricket 
+3. Piano
+4. Learning New and Exciting Things
+
+# **EDUCATION**
+
+Class 1-12 -- Essar International School
```

Comment:
I think whatever you were trying to do with `--` didn't work? You can look at your current md file [here](https://github.com/ACM-IGDTUW/Open-Source-Challenge/blob/4262b6bc17d9ba5c4c769fdd36ea6a24042f8d46/participants/sanjitcodes.md).